tests/tabletoolbar.js

```diff
@@ -239,12 +239,11 @@ describe( 'TableToolbar', () => {
 	} );
 
 	describe( 'deprecated toolbar', () => {
```

Comment:
To be honest I don't think we need unit tests for removing a variable. Instead we should remove the suite all along.